stl/inc/xutility

```diff
@@ -34,6 +34,8 @@ __declspec(noalias) void __cdecl __std_reverse_trivially_swappable_1(void* _Firs
 __declspec(noalias) void __cdecl __std_reverse_trivially_swappable_2(void* _First, void* _Last) noexcept;
 __declspec(noalias) void __cdecl __std_reverse_trivially_swappable_4(void* _First, void* _Last) noexcept;
 __declspec(noalias) void __cdecl __std_reverse_trivially_swappable_8(void* _First, void* _Last) noexcept;
+__declspec(noalias) void __cdecl __std_swap_ranges_trivially_swappable_noalias(
+    void* _First1, void* _Last1, void* _First2) noexcept;

```

Comment:
In `<algorithm>`, this declaration was additionally guarded by `&& !defined(_M_HYBRID)`. Should that be retained here? I see that the usage below is still guarded.

Reply:
Hmmm I suspect that's an overabundance of paranoia but will check it out.

Reply:
I'm not very concerned about having a declaration that isn't used; feel free to wontfix this.

Reply:
Yeah, but it probably means the guard is broken and should be removed. I can't think of a reason we would want to guard swap_ranges but not reverse or vice versa; and reverse's call is not guarded for hybrid.

Reply:
Ah, good catch.